pkg/debugapi/chequebook.go

```diff
@@ -244,6 +244,50 @@ func (s *Service) swapCashoutHandler(w http.ResponseWriter, r *http.Request) {
 	jsonhttp.OK(w, swapCashoutResponse{TransactionHash: txHash.String()})
 }
 
+func (s *Service) swapReCashoutHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
since the logic is almost identical with `swapCashoutHandler` why don't you add another field in the request for redo rather than defining a new method?